ochre/Equipment/WaterHeater.py

```diff
@@ -424,25 +428,38 @@ def __init__(self, hp_only_mode=False, water_nodes=12, **kwargs):
         self.deadband_temp = kwargs.get('Deadband Temperature (C)', 8.17)  # different default than ERWH
 
         # Nominal COP based on simulation of the UEF test procedure at varying COPs
-        self.cop_nominal = kwargs['HPWH COP (-)']
-        self.hp_cop = self.cop_nominal
-        if self.cop_nominal < 2:
-            self.warn("Low Nominal COP:", self.cop_nominal)
-
-        # Heat pump capacity and power parameters - hardcoded for now
-        if 'HPWH Capacity (W)' in kwargs:
-            self.hp_capacity_nominal = kwargs['HPWH Capacity (W)']  # max heating capacity, in W
-        else:
-            hp_power_nominal = kwargs.get('HPWH Power (W)', 500)  # in W
+        if self.low_power_hpwh: #TODO: can read a lot of these directly when properly integrated into HPXML
+            self.cop_nominal = 4.2
+            self.hp_cop = self.cop_nominal
+            self.hp_capacity_nominal = 1499.4
+            hp_power_nominal =  self.hp_capacity_nominal / self.cop_nominal # in W
             self.hp_capacity_nominal = hp_power_nominal * self.hp_cop  # in W
-        self.hp_capacity = self.hp_capacity_nominal  # in W
+            self.hp_capacity = self.hp_capacity_nominal  # in W
+        else:
+            self.cop_nominal = kwargs['HPWH COP (-)']
+            self.hp_cop = self.cop_nominal
+            if self.cop_nominal < 2:
+                self.warn("Low Nominal COP:", self.cop_nominal)
+
+            # Heat pump capacity and power parameters - hardcoded for now
+            if 'HPWH Capacity (W)' in kwargs:
+                self.hp_capacity_nominal = kwargs['HPWH Capacity (W)']  # max heating capacity, in W
+            else:
+                hp_power_nominal = kwargs.get('HPWH Power (W)', 500)  # in W
+                self.hp_capacity_nominal = hp_power_nominal * self.hp_cop  # in W
+            self.hp_capacity = self.hp_capacity_nominal  # in W
         self.parasitic_power = kwargs.get('HPWH Parasitics (W)', 1)  # Standby power in W
         self.fan_power = kwargs.get('HPWH Fan Power (W)', 35)  # in W
 
         # Dynamic capacity coefficients
         # curve format: [1, t_in_wet, t_in_wet ** 2, t_lower, t_lower ** 2, t_lower * t_in_wet]
-        self.hp_capacity_coeff = np.array([0.563, 0.0437, 0.000039, 0.0055, -0.000148, -0.000145])
-        self.cop_coeff = np.array([1.1332, 0.063, -0.0000979, -0.00972, -0.0000214, -0.000686])
+        if self.low_power_hpwh:
+            self.hp_capacity_coeff = np.array([0.813, 0.0160, 0.000537, 0.0020319, -0.0000860, -0.0000686])
+            self.cop_coeff = np.array([1.1332, 0.063, -0.0000979, -0.00972, -0.0000214, -0.000686])
```

Comment:
Can you double check this? Looks like you copied the existing capacity values
to the normal HPWH, but the existing COP values to the low power HPWH.

Reply:
Good call, but this is actually up to date (ie different) in the latest version? 

https://github.com/NREL/OCHRE/blob/b781132a77e95ccfcd355dab608ec63cb5bf9c6a/ochre/Equipment/WaterHeater.py#L462